renku_notebooks/api/custom_fields.py

```diff
@@ -59,3 +59,7 @@ def _deserialize(
     is not None,
     required=True,
 )
+serverOptionUrlValue = fields.Str(
+    required=True,
+    validate=lambda x: re.match(r"^\/[a-zA-Z0-9\-\.\_\~\/]+", x) is not None,
```

Comment:
Not sure if this is working as expected. I can set a default to `/tree!!!` and it doesn't fail. I assume it's matching the first part, so `is not None` evaluates to `True`. Is that intended?

Reply:
Here is an example https://tasko.dev.renku.ch/projects/lorenzo.cavazzi.tech/new-project-test/environments

Reply:
my bad, this should fix it. `"^\/[a-zA-Z0-9\-\.\_\~\/]+$"`. I am redeploying.